recipes/metaknowledge/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "metaknowledge" %}
+{% set version = "3.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f137dce7ed7e1fb3a73730716d37e2b5757d9fc94f3bb50f8e82338c38b17dae
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - metaknowledge = metaknowledge.bin:mkCLI
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
I think you only need `python` and `pip` in the host requirements section. These are just the requirements used for running the `setup.py` file not those used by the package itself.

Reply:
Ah yes, indeed. I think I should include the others in the `test` section as `requires` though.

Reply:
Or is that already implied by the `run` requirements actually?

Reply:
Yes they will be.

`test` requirements are for things that are only needed when testing (like `pytest` or `nose`).

Reply:
Ah yes, of course, thanks.